docs/sources/reference/commandline/cli.md

```diff
@@ -847,6 +848,11 @@ you refer to it on the command line.
 > children) for security reasons, and to ensure repeatable builds on remote
 > Docker hosts. This is also the reason why `ADD ../file` will not work.
 
+When `docker build` is run with the `--cgroup-parent` option the containers used
+in the build will be run with the [corresponding `docker run`
+flag](/reference/run/#specifying-custom-cgroups). 
```

Comment:
This PR adds a little documentation to that run section which, while admittedly light on coverage, is better than nothing.